feature-libs/checkout/base/components/checkout-delivery-mode/checkout-delivery-mode.component.ts

```diff
@@ -100,6 +101,33 @@ export class CheckoutDeliveryModeComponent {
     protected activeCartFacade: ActiveCartFacade
   ) {}
 
+  onKeydown(code: string | undefined, event: Event): void {
```

Comment:
Could we please have some comments or methods to outline why we are capturing the keydown event?

Reply:
Comment added for clarity.